maskrcnn_benchmark/engine/trainer.py

```diff
@@ -101,8 +101,10 @@ def do_train(
                     memory=torch.cuda.max_memory_allocated() / 1024.0 / 1024.0,
                 )
             )
-        if iteration % checkpoint_period == 0 or iteration == max_iter:
+        if iteration % checkpoint_period == 0:
```

Comment:
Is this change intended?

Reply:
There was a merge error before, and it is fixed now.